introduction.md

```diff
@@ -1,2 +1,2 @@
-This file is created for training purpose. Code School helps to learn Ruby, JavaScript, HTML, ...
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?